composer.json

```
@@ -188,7 +188,7 @@
         "api-platform/core": "^2.5.5",
         "doctrine/inflector": "^1.4",
         "sylius/grid-bundle": "1.7.4",
-        "symfony/form": "4.4.9",
+        "symfony/form": "^4.4.9",
```

Maybe we should add:
```suggestion
        "symfony/form": "4.4.9|4.4.10",
```